src/pybind11/test/test_frustum.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
I think the best strategy for the test is to duplicate src/test/PyImathTest/pyImathTest.in in src/pybind11, and comment out the sections that aren't yet implemented, and then bring the back in as new classes/features get implemented. That's better than building up a duplicate set of tests. I recommend just removing this for now and dealing with the test as a separate PR

Reply:
> I recommend just removing this for now and dealing with the test as a separate PR

I will remove those tests - those were sanity check for me - that the code actually runs. The code can  compile, but we might get a lot of runtime issues related to unbinded types, etc.

> That's better than building up a duplicate set of tests.

Agreed. I need to spend more time to understand what was the premise of making integration tests as `*.in` file, why not break it down into multiple different files. Ideally, we would like to run same tests for `boost` and `pybind11` to confirm that we are backwards compatible. 

Can you share what is the future of `boost` bindings? Would you like to remove them?

Reply:
The plan is to ultimately retire the boost bindings altogether, once the pybind implementation replicates all the functionality. The setup of building both in parallel allows us to implement the pybind wrappings incrementally. I'll look into arranging the test to run with the pybind implementation. I agree the test would be more manageable as separate files, but that's the way it was implemented.

Reply:
One more question related to `*Array` classes, what was motivation for those? Perf? Would you like to transition them to pybind11 in the future or retire them as well?